src/api/Nhs.Appointments.Api/Functions/NotifyUserRolesChangedFunction.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Threading.Tasks;
+using MassTransit;
+using Microsoft.Azure.WebJobs;
+using Azure.Messaging.ServiceBus;
+using System.Threading;
+using Nhs.Appointments.Api.Consumers;
+
+namespace Nhs.Appointments.Api.Functions;
+
+public class NotifyUserRolesChangedFunction
+{
+    const string UserRolesChangedQueueName = "user-role-change";
+    readonly IMessageReceiver _receiver;
+
+    public NotifyUserRolesChangedFunction(IMessageReceiver receiver)
```

Comment:
Use a primary constructor (also removes the need for the private member variable)